.travis.yml

```diff
@@ -10,7 +10,10 @@ language: node_js
 
 node_js:
   - 8
-  
+
+install:
+  - npm install jsdom
```

Comment:
Is is possible to use `jsdom` [webjars](https://www.webjars.org/) managed by sbt instead of external command `npm install`, preventing tests running failed when one who checkout this repository and simply run `sbt test`.

Reply:
I think the normal way is described here: https://www.scala-js.org/doc/project/js-environments.html

> Scala.js 1.x: The above setting requires the following line in your project/plugins.sbt:
> libraryDependencies += "org.scala-js" %% "scalajs-env-jsdom-nodejs" % "1.0.0-M1"

Reply:
So what's the normal way to create a library for both Scala.js versions?

Reply:
I think you need something like this in your plugins.sbt: 
```scala
addSbtPlugin("org.scala-native"  % "sbt-crossproject"         % "0.2.2")
addSbtPlugin("org.scala-native"  % "sbt-scala-native"         % "0.3.3")

// For sbt-crossproject support even with Scala.js 0.6.x
{
  if (scalaJSVersion.startsWith("0.6."))
    Seq(addSbtPlugin("org.scala-native" % "sbt-scalajs-crossproject" % "0.2.2"))
  else
    Nil
}
```

Reply:
This repository use sbt-release, which is not used by scalatags. I don't think sbt-release is able to switch other sbt plugins.

Reply:
I am not sure yet, I try to figure it out but utest also allows cross version support and after introducing cross-platform support they switched from sbt-release to sbt-dynver:
https://github.com/lihaoyi/utest/commit/47a5ecf5fa5fccfff81a80c276609206a5e7a6e4#diff-a84f91f20f040218bccd09fed4761fb3